File: testdata/Casts.java
import java.util.Locale;

class Casts
{
  void yo(Object arg)
  {
    String casted = (String) arg;
    Object original = (java.lang.Object) casted;
  
    System.out.println(arg.toString().toUpperCase(Locale.US));
  
    float f = 4.5f;
    int i = (int) f;
    long l = (long) i;
    double d = (double) f;
    float f2 = (float) d;
    byte b = (byte) i;
    short s = (short) i;    
  }
}

File: testdata/expected/default/Casts.scala
import java.util.Locale

class Casts
{
  def
  yo
  (
    arg : Object
  ) : Unit =
  {
    var casted : String = arg.asInstanceOf[String]
    var original : Object = casted.asInstanceOf[java.lang.Object]
  
    System.out.println(arg.toString().toUpperCase(Locale.US))
  
    var f : Float = 4.5f
    var i : Int = f.toInt
    var l : Long = i.toLong
    var d : Double = f.toDouble
    var f2 : Float = d.toFloat
    var b : Byte = i.toByte
    var s : Short = i.toShort    
  }
}